VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -0,0 +1,83 @@
+// FriendListTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+typealias TapHandler = ((NetworkUnit) -> ())
+
+/// Экран со списком друзей.
+final class FriendListTableViewController: UITableViewController {
+
+    // MARK: - Private properties
+
+    private var users: [User] = []
+    private var selectedFriend: NetworkUnit?
+    private var tapHandler: TapHandler?
+
+    // MARK: - Life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        makeUsers()
+        regCells()
+        configreTapHandler()
```

Comment:
```suggestion
       setupCell()
```
  
  
  func setupCell {
      makeUsers()
        regCells()
        configreTapHandler()
        }